### ISSUE TITLE
Too Many Directories

### ISSUE THREAD

**Issue by ghost** [spritebuilder/SpriteBuilder#227]
Just received this message. 

![screen shot 2014-01-28 at 11 37 25 am](https://f.cloud.github.com/assets/5808253/2021141/adc5c57e-883a-11e3-9fa4-103a6e1e0d75.png)

If I remove one directory it goes away so it definitely seems like an arbitrary value is being tested on. Is this a performance restriction? Is there anyway to increase the number of folders?

**Comment by miyahira** [comment]
The limit is defined in https://github.com/apportable/SpriteBuilder/blob/6ba7e2a0bce55053cdada82f7f99bb8a392b9052/SpriteBuilder/ccBuilder/ResourceManager.h#L29 , so I just git-cloned and been using my custom version of SpriteBuilder(and CocosBuilder) for a while,

but I would also be happy to be able to override this limit in the settings window or somewhere.

**Comment by bchessin** [comment]
Yea can this actually be fixed? I am not even sure why there is a folder limit. It just makes designing apps take longer to do now that it requires me to use 2 separate CocosBuilder projects.

**Comment by NickyWeber** [comment]
@vlidholt we can make the limit a configuration option for the time being, so users can raise the value at their own risk.

**Comment by bchessin** [comment]
@NickyWeber Yes please do! If/when you update SpriteBuilder with this, please let me know!
